test/OrchardCore.Tests/Scripting/ScriptFunctionsTest.cs

```diff
@@ -0,0 +1,34 @@
+using System.Text.Json.Nodes;
+using OrchardCore.Scripting;
+using OrchardCore.Tests.Apis.Context;
+
+namespace OrchardCore.Tests.Scripting;
+public class ScriptFunctionsTest
+{
+    [Fact]
+    public async Task TheScriptingEngineShouldBeAbleToHandleJsonObject()
+    {
+        using var context = new SiteContext();
+        await context.InitializeAsync();
+        await context.UsingTenantScopeAsync(scope =>
+        {
+            var findUser = new GlobalMethod
+            {
+                Name = "tryAccessJsonObject",
+                Method = sp => () =>
+                {
+                    return new JsonObject
```

Comment:
Could you try adding other STJ types in the test, like arrays, bool, numbers, to be sure it works fine too.

Reply:
the jint project contains the full test, do I need to copy it to oc?

https://github.com/sebastienros/jint/blob/107f5433293e60dfc96f0bff83c427c6d7081537/Jint.Tests.PublicInterface/InteropTests.SystemTextJson.cs#L32-L59

Reply:
The problem is that false value in JsonObject does not work as expected 

Hello, thanks for the PR, but I find that false for Boolean values doesn't seem to handle as expected

Reply:
relate https://github.com/sebastienros/jint/issues/1801